src/wf_utils/labels_tw.config

```diff
@@ -14,6 +17,11 @@ process {
   withLabel: lowmem { memory = { get_memory( 20.GB * task.attempt ) } }
   withLabel: midmem { memory = { get_memory( 50.GB * task.attempt ) } }
   withLabel: highmem { memory = { get_memory( 100.GB * task.attempt ) } }
+
+  withLabel: lowdisk { disk = 100.GB }
+  withLabel: middisk { disk = 250.GB }
+  withLabel: highdisk { disk = 500.GB }
```

Comment:
Should disk also scale with attempt #? :thinking: 

E.g. attempt 1: 50GB, attempt 2: 200 GB, attempt 3: 500 GB?

Reply:
was thinking the same thing

Reply:
The issue at hand is that these processes fail with error code 1 so no real way of detecting that the disk storage is the problem.